### PR TITLE
Feature/sidenav

### PR DESCRIPTION
**Pull request by VijuMadathil**
Updated the sidenav component, now sidenav is a reusable seprate component which accept the json as input parameter. Json file should contain the Side Nav contents and path URL

### CODE REVIEW

File: src/app/featureModules/admin/components/dashboard/dashboard.component.ts [hunk 1]
```diff
@@ -10,12 +11,18 @@ import { map, shareReplay } from 'rxjs/operators';
 })
 export class DashboardComponent {
 
+  public shosideNavEle = true;
```
Comment (jessin08) [inline src/app/featureModules/admin/components/dashboard/dashboard.component.ts:14]: public shosideNavEle = true; it should be showSideNav

File: src/app/featureModules/admin/components/dashboard/dashboard.component.ts [hunk 2]
```diff
@@ -10,12 +11,18 @@ import { map, shareReplay } from 'rxjs/operators';
...
     .pipe(
       map(result => result.matches),
       shareReplay()
     );
+    
+  sideNavData: any  = (sideNavData  as  any).default;
+  constructor(private breakpointObserver: BreakpointObserver) {
 
-  constructor(private breakpointObserver: BreakpointObserver) {}
+    console.log('dashboard');
```
Comment (jessin08) [inline src/app/featureModules/admin/components/dashboard/dashboard.component.ts:24]: Need to remove console.log

File: src/app/featureModules/admin/components/dashboard/dashboard.component.ts (hunk 1 above)
Comment (jessin08) [inline src/app/featureModules/admin/components/dashboard/dashboard.component.ts:14]: again.. shosideNavEle should be showSideNav

File: src/app/featureModules/admin/components/dashboard/dashboard.component.ts (hunk 2 above)
Comment (jessin08) [inline src/app/featureModules/admin/components/dashboard/dashboard.component.ts:24]: remove console.log

**Comment by VijuMadathil** [comment]
Updated Sidevan as Reusable component

**Pull request merged**